test/cypress/components/multiselect.js

```diff
@@ -0,0 +1,91 @@
+export class Multiselect {
+
+  click() {
+    return cy.get( '.o-expandable_target' ).click();
+  }
+
+  multiSelect( name ) {
+    return cy.get( `.o-multiselect_${ name }` ).first();
+  }
+
+  choices() {
+    return this.multiSelect( 'choices' );
+  }
+
+  header() {
+    return this.multiSelect( 'header' );
+  }
+
+  searchInput() {
+    return this.multiSelect( 'search' );
+  }
+
+  fieldSet() {
+    return this.multiSelect( 'fieldset' );
+  }
+
+  optionsField() {
+    return this.multiSelect( 'options' );
+  }
+
+  clickSearchInput() {
+    return this.searchInput().click( { force: true } );
+  }
+
+  enterSearchInput( name ) {
+    return this.searchInput().clear().type( name );
+  }
+
+  selectedTags = [];
```

Comment:
Hmm the linter is breaking down on some of this OOP syntax. Not sure if the linter is wrong or if this should be in a constructor? 

Run `gulp lint:tests --fix` and scroll up to the error to see (will also fix other minor formatting issues).

Reply:
I fixed most minor formatting issues with spacing and comments.
I do not believe the linter is wrong so I added a constructor to the Multiselect class.
Would other classes benefit from a constructor?

Reply:
I think only if the linter complains about it or for consistency-sake you think it would be worth it.